container_workflow_tool/distgit.py

```diff
@@ -100,40 +100,56 @@ def _bump_release(self, version_str, bump_type):
                 length[-1] = str(int(length[-1]) + 1)
             return '.'.join(length)
 
-    def _get_from(self, dockerfile_path):
+    def _get_tag(self):
+        return 'rhel' + self.base_image.split(':')[1].split('-')[0]
```

Comment:
I do not think we need rhel in Fedora :)

Reply:
function `_get_tag` is not used at all ;)

Reply:
Let's get rid of it by the next commit.